logicmonitor/resource_device.go

```diff
@@ -160,3 +164,50 @@ func deleteDevice(d *schema.ResourceData, m interface{}) error {
 	d.SetId("")
 	return nil
 }
+
+// function to import devices
+func resourceDeviceStateImporter(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+	client := m.(*lmv1.DefaultApi)
+
+	// if it is a deviceId, we will add the group directly
+	if checkID(d.Id()) {
```

Comment:
is it normal provider practice accept two different possible types of resource identifiers? it's definitely convenient, just not sure if it's a normal best practice.

Reply:
im not too sure. I looked for a bit for provider best practices documentation. I know ID is usually standard but since LM doesn't make that easy for users (unless you run api calls or dig deep into the info tab), I added that second option. Figured itd add more benefit than harm

Reply:
Makes sense. In that case, I'd probably be in favor of ditching the ID and just using the display name. No strong preference either way though.